File: library/src/main/java/com/example/library/dto/response/MemberSyncResponse.java
package com.example.library.dto.response;


import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class MemberSyncResponse {

    private Integer id;

    private String cardNumber;

    private String name;

    private Integer createdBy;
    
    private Long version;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt; 

}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/MemberSyncResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDateTime

data class MemberSyncResponse(
    val id: Int,
    val cardNumber: String,
    val name: String,
    val createdBy: Int,
    val updatedAt: LocalDateTime,
    val version : Long,
    val deleteAt: LocalDateTime?
)